cg/src/domains/processors/processors.routes.js

```diff
@@ -20,7 +25,9 @@ module.exports = app => {
   );
 
   router.get(
-    '/contract/details',
-    asyncHandler(async (req, res) => processorsController.getContractDetails(req, res))
+    '/subject/:subjectId/get-restrictions',
+    controllerOnly,
+    asyncHandler(ensureProcessorAccessToSubject),
+    asyncHandler(async (req, res) => processorsController.getSubjectRestrictions(req, res))
```

Comment:
I think we can make it shorter here
```
asyncHandler(processorsController.getSubjectRestrictions)
```

Reply:
Gonna take a look at it

Reply:
We actually can't, the method getSubjectRestrictions needs to know it's input parameters. 
We can use this shorthand when the middleware does not need any parameter